src/explore-education-statistics-frontend/src/modules/find-statistics/FindStatisticsPageNew.tsx

```diff
@@ -0,0 +1,155 @@
+import LoadingSpinner from '@common/components/LoadingSpinner';
+import RelatedInformation from '@common/components/RelatedInformation';
+import { PublicationSummaryWithRelease } from '@common/services/publicationService';
+import { Paging } from '@common/services/types/pagination';
+import Page from '@frontend/components/Page';
+import Pagination from '@frontend/components/Pagination';
+import useRouterLoading from '@frontend/hooks/useRouterLoading';
+import PublicationSummary from '@frontend/modules/find-statistics/components/PublicationSummary';
+import { NextPage } from 'next';
+import React, { useEffect, useState } from 'react';
+
+interface Props {
+  paging: Paging;
+  publications: PublicationSummaryWithRelease[];
+}
+
+const FindStatisticsPageNew: NextPage<Props> = ({ paging, publications }) => {
+  // TODO EES-3517 - update totalResults when search / filter
+  const { totalResults } = paging;
+  // const [totalResults, setTotalResults] = useState<number>(paging.totalResults);
+  const [currentPage, setCurrentPage] = useState<number>(paging.page);
+  const [currentPublications, setCurrentPublications] = useState<
+    PublicationSummaryWithRelease[]
+  >(publications);
+
+  useEffect(() => {
+    setCurrentPage(paging.page);
+    setCurrentPublications(publications);
+  }, [paging.page, publications]);
```

Comment:
We shouldn't need any of this if all paging is being handled by `getServerSideProps`. Prop changes will re-render this component so we're currently just copying props into state (which is redundant).